apps/simple-trading-app/src/app/components/simple-market-list/simple-market-list.spec.tsx

```diff
@@ -130,8 +137,12 @@ describe('SimpleMarketList', () => {
       await new Promise((resolve) => setTimeout(resolve, 0));
     });
     await waitFor(() => {
-      expect(screen.getByTestId('simple-market-list')).toBeInTheDocument();
+      expect(
+        document.querySelector('.ag-center-cols-container')
```

Comment:
question (non-blocking): this seems to be testing a third party implementation, as this className gets added by AgGrid, if I'm not mistaken. not sure if relying on the third party class name is better here opposed to the previous version of having an actual test id and testing if that is in the document.

Reply:
I got your point, but here we need to wait for and catch a moment when agGrid is rendered, and since it not accept param as `data-testid`, relaying on catching this would be ineffective.
I think if they would change a name of the css class it would be a breaking change, and the correspondingly bumped version number would not let for an automatic uncontrolled upgrade of the package and all the problems come with it.
Other thing, their [blog](https://blog.ag-grid.com/using-react-testing-library-with-ag-grid/) also recommends smth similar..